.circleci/config.yml

```diff
@@ -0,0 +1,58 @@
+
+version: 2
+
+defaults: &defaults
+  docker:
+    - image: circleci/ruby:2.3
```

Comment:
@namrataukirde  I think it would be fine to use the latest ruby image and update the Gemfile / capistrano version etc. Best to go with 2.6 now I think